src/test/CheckResultsByLevel.test.tsx

```diff
@@ -0,0 +1,354 @@
+    const checkHeader = check.find("p.MuiTypography-root");
+    expect(checkHeader.length).toEqual(1);
+    expect(checkHeader.at(0).text()).toEqual("Status: pending • Service Ownership check");
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
+    const checkSubheader = checkHeader.find("span");
+    expect(checkSubheader.length).toEqual(1);
+    expect(checkSubheader.at(0).text()).toEqual(" • Service Ownership check");
+    expect(checkSubheader.at(0).props().className).toContain("coloredSubtext");
+    const checkContents = check.find("div.MuiAccordionDetails-root");
+    expect(checkContents.length).toEqual(1);
+    expect(checkContents.at(0).find(".p-will-be-enabled").props().hidden).toBe(true);
+    expect(checkContents.at(0).find(".p-check-message").text()).toEqual("Error: This check has status pending.");
+    expect(checkContents.at(0).find(".p-last-updated").props().hidden).toBe(true);
+  });
+
+  it('renders a passed check appropriately', () => {  
+    const wrapper = getWrapper(checkResultsByLevelData, 4, 2);
+
+    const checkCandidates = wrapper.find("div#accordion-check-3");
+    expect(checkCandidates.length).toEqual(1);
+    const check = checkCandidates.at(0);
+    expect(check.props().style?.backgroundColor).toEqual("#00ff0055");
+    const checkHeader = check.find("p.MuiTypography-root");
+    expect(checkHeader.length).toEqual(1);
+    expect(checkHeader.at(0).text()).toEqual("Status: passed • Uncategorized check");
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
+    const checkSubheader = checkHeader.find("span");
+    expect(checkSubheader.length).toEqual(1);
+    expect(checkSubheader.at(0).text()).toEqual(" • Uncategorized check");
+    expect(checkSubheader.at(0).props().className).toContain("coloredSubtext");
+    const checkContents = check.find("div.MuiAccordionDetails-root");
+    expect(checkContents.length).toEqual(1);
+    expect(checkContents.at(0).find(".p-will-be-enabled").props().hidden).toBe(true);
+    expect(checkContents.at(0).find(".p-check-message").text()).toEqual("Error: This check has status passed.");
+    expect(checkContents.at(0).find(".p-last-updated").props().hidden).toBe(false);
+    expect(checkContents.at(0).find(".p-last-updated").text()).toEqual("Last updated: May 11th 2023, 20:47:53 (UTC)");
+  });
+
+  it('renders an upcoming_failed check appropriately', () => {  
+    const wrapper = getWrapper(checkResultsByLevelData, 4, 2);
+
+    const checkCandidates = wrapper.find("div#accordion-check-4");
+    expect(checkCandidates.length).toEqual(1);
+    const check = checkCandidates.at(0);
+    expect(check.props().style?.backgroundColor).toEqual("#00000055");
+    const checkHeader = check.find("p.MuiTypography-root");
+    expect(checkHeader.length).toEqual(1);
+    expect(checkHeader.at(0).text()).toEqual("Status: upcoming_failed • Service Ownership check");
+    expect(checkHeader.at(0).props().className).toContain("coloredText");
```

Comment:
this sort of thing doesn't belong in a test case. Our tests should assert text, buttons, etc. If we change styles, I don't think that should break any tests.

Reply:
isn't the coloring based on status part of the logic of this change?